sdk/core/azure-core/HISTORY.md

```diff
@@ -7,6 +7,7 @@
 
 - Support auto_request_id in RequestIdPolicy   #9163
 - Support fixed retry   #6419
+- Support "retry-after-ms" in response header   #9240
```

Comment:
This is only relevant to appconfig? Would add "for appconfig" to the issue title and here to give anyone reading the context for the change.

Reply:
Last time when I asked Johan, he told me we should support "retry-after-ms" regardless which service uses it.